src/utils/composables/useMovableElement.composable.js

```diff
@@ -9,16 +9,16 @@ const defaultPadding = 4 // px
  * It is possible to specify custom offset with screen border, in order to avoid some part of the
  * app (such as the header) to be accessible to the movable element
  *
- * @param element
+ * @param element Reference to an DOM element
```

Comment:
small typo here `Reference to a DOM element` (DOM might have been added later :wink: )